src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -73,6 +73,7 @@
             "List", "ArrayList", "Deque", "Queue", "LinkedList",
             "Set", "HashSet", "SortedSet", "TreeSet",
             "Map", "HashMap", "SortedMap", "TreeMap",
+            "Override", "Deprecated", "SafeVarargs", "SupressWarnings",
```

Comment:
@rnveach , do you think we need to add all other basic standard annotations ?

Reply:
which annotations are you talking about? I can't think of any others but these.

Reply:
https://docs.oracle.com/javase/8/docs/api/java/lang/package-summary.html
Section "Annotation Types Summary".

`FunctionalInterface` should be added to be default.

Reply:
fixed